tests/scenarios/catchup-with-indexer.spec.ts

```diff
@@ -38,8 +38,8 @@ describe('Subscribing using catchup-with-indexer', () => {
     const { txns } = await SendXTransactions(5, testAccount, algorand)
     const { lastTxnRound, txIds } = await SendXTransactions(1, randomAccount, algorand)
     await waitForIndexerTransaction(txIds[0])
-    const expectedNewWatermark = Number(txns[2].confirmation!.confirmedRound!) - 1
-    const indexerRoundsToSync = expectedNewWatermark - initialWatermark
+    const expectedNewWatermark = txns[2].confirmation!.confirmedRound! - 1n
+    const indexerRoundsToSync = Number(expectedNewWatermark - initialWatermark)
```

Comment:
Optional: to avoid this awkardness we could allow the range configurations to be number or bigint?